rust/src/connection/network/stub.rs

```diff
@@ -132,6 +148,17 @@ impl<Channel: GRPCChannel> RPCStub<Channel> {
         self.single(|this| Box::pin(this.grpc.database_type_schema(req.clone()))).await
     }
 
+    pub(super) async fn database_export(
+        &mut self,
+        req: database::export::Req,
+    ) -> Result<Streaming<database::export::Server>> {
+        self.call_with_auto_renew_token(|this| {
+            let req = req.clone();
```

Comment:
Is this necessary? Other requests get away with inlining `req.clone()` directly in the closure.